FirebaseAI/Sources/Types/Internal/ImageAPIMethod.swift

```diff
@@ -0,0 +1,18 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+enum ImageAPIMethod: String {
+  case generateImages = "templatePredict"
+}
```

Comment:
I think this should be nested inside `TemplateGenerateImagesRequest` since it's very specific to SPT.